packages/shared/src/components/squads/settings/SquadModerationSettingsSection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ReactElement, useState } from 'react';
+import { SquadSettingsSection } from './SquadSettingsSection';
+import { Radio } from '../../fields/Radio';
+import { SourceMemberRole } from '../../../graphql/sources';
+import { Switch } from '../../fields/Switch';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { SimpleTooltip } from '../../tooltips';
+import { useToggle } from '../../../hooks/useToggle';
+
+interface SquadModerationSettingsSectionProps {
+  initialMemberPostingRole?: SourceMemberRole;
+  initialMemberInviteRole?: SourceMemberRole;
+  initialModerationRequired?: boolean;
+}
+
+const memberRoleOptions = [
+  {
+    label: 'All members (recommended)',
+    value: SourceMemberRole.Member,
+  },
+  {
+    label: 'Only moderators',
+    value: SourceMemberRole.Moderator,
+  },
+];
+
+export function SquadModerationSettingsSection({
+  initialMemberInviteRole,
+  initialMemberPostingRole,
+  initialModerationRequired,
+}: SquadModerationSettingsSectionProps): ReactElement {
+  const [isHoveringSwitch, setIsHoveringSwitch] = useState(false);
+  const [memberPostingRole, setMemberPostingRole] = useState(
+    () => initialMemberPostingRole || SourceMemberRole.Moderator,
```

Comment:
> IMO the initial state should not be a function, just the value is fine!

This is only showing up as a change since I renamed the component, as it was like this from before, but I can do a little refactoring while I'm at it :)